test/test_ops.py

```diff
@@ -574,11 +574,16 @@ def test_slice_in_bounds_multidim(self):
     helper_test_op([(3,3,3)], lambda x: x[1:2, 1:2], lambda x: x[1:2, 1:2])
     helper_test_op([(3,3,3)], lambda x: x[1:2, 1:2, 0:-1], lambda x: x[1:2, 1:2, 0:-1])
 
-  def test_slice_with_none(self):
+  def test_slice_with_none_and_npnewaxis(self):
```

Comment:
Why is np.newaxis supported? This should be the same as None

Reply:
`  # - Indexing with np.newaxis or None on a given axis will add a new dimension of size one before that axis`
this line in the description of `_getitem_` mentioned npnewaxis and I didn't see it tested xD

But `np.newaxis` just evaluates to `None` when passed in as arg to `_getitem_`, so really they literally are the same thing.
Yeah, you're right, I'll also del the np.newaxis in the description. 
People should just use None instead.